### PR TITLE
Remove collections model from snoop2

### PR DESCRIPTION
**Pull request by mgax**
This patch removes the concept of collections from snoop2. Before updating production servers, make sure of the following:

* make sure there is only one collection in the snoop, otherwise the DB migration will fail
* configure the `SNOOP_COLLECTIONS_ELASTICSEARCH_INDEX ` setting to the name of the elasticsearch index
* configure the `SNOOP_COLLECTION_ROOT` setting to the path of files in the collection
* update URLs in hoover-search - the collection json URL is now `http://$SNOOP_SERVER/collection/json`

The PR is work in progress; remaining tasks:
* [ ] fix tests

### CODE REVIEW

File: snoop/data/exportimport.py [hunk 1]
```diff
@@ -27,172 +25,15 @@
...
-            Q(pk__in=tasks.values_list('result_id', flat=True)) |
-            Q(pk__in=tasks.values_list('blob_arg_id', flat=True))
-        )
-    )
-
-    task_dependencies = (
-        models.TaskDependency.objects
-        .filter(next__in=tasks)
-    )
+def build_export_queries():
```
Comment (mugurrus) [inline snoop/data/exportimport.py]: This is not needed anymore. The database export will be done by copying the db directory.

File: snoop/data/exportimport.py [hunk 2]
```diff
@@ -203,40 +44,14 @@ def build_export_queries(collection):
 
 
 @transaction.atomic
-def export_db(collection_name, stream=None):
-    collection = models.Collection.objects.get(name=collection_name)
+def export_db(stream=None):
```
Comment (mugurrus) [inline snoop/data/exportimport.py]: Same for this.

File: snoop/data/exportimport.py [hunk 3]
```diff
@@ -304,14 +119,12 @@ def pk_interval(queryset):
...
-    log.info("Exporting '{}' took {:1.2f}s".format(collection, time.perf_counter()-start))
+    log.info("Exporting took {:1.2f}s".format(time.perf_counter()-start))
 
 
 @transaction.atomic
-def import_db(collection_name, stream=None):
-    collection = models.Collection.objects.create(name=collection_name,
-                                                  root='')
-    log.info("Importing %r", collection)
+def import_db(stream=None):
```
Comment (mugurrus) [inline snoop/data/exportimport.py]: Not needed anymore.

File: snoop/data/exportimport.py [hunk 4]
```diff
@@ -449,9 +259,8 @@ def load_file(name, pk_delta=None):
                       duplicate_task_dependencies)
 
 
-def export_blobs(collection_name, stream=None):
-    collection = models.Collection.objects.get(name=collection_name)
-    queries = build_export_queries(collection)
+def export_blobs(stream=None):
```
Comment (mugurrus) [inline snoop/data/exportimport.py]: Not needed

File: snoop/data/management/commands/exportdb.py [hunk 5]
```diff
@@ -0,0 +1,11 @@
+from django.core.management.base import BaseCommand
+from ...logs import logging_for_management_command
+from ... import exportimport
+
+
+class Command(BaseCommand):
```
Comment (mugurrus) [inline snoop/data/management/commands/exportdb.py]: Not needed

File: snoop/data/management/commands/importdb.py [hunk 6]
```diff
@@ -0,0 +1,11 @@
+from django.core.management.base import BaseCommand
+from ...logs import logging_for_management_command
+from ... import exportimport
+
+
+class Command(BaseCommand):
```
Comment (mugurrus) [inline snoop/data/management/commands/importdb.py]: Not needed

**Comment by mgax** [comment]
Tests pass. Next I will remove the management commands.

Before we merge this, we should also update the scripts in `docker-setup`, @mugurrus could you look into that, since you know those scripts better?

**Pull request merged**